_topic_maps/_topic_map_ms.yml

```diff
@@ -113,6 +113,40 @@ Distros: microshift
 Topics:
 - Name: Understanding networking
   File: microshift-networking
+---
+Name: Storage
+Dir: microshift_storage
+Distros: microshift
+Topics:
+- Name: MicroShift storage overview
+  File: microshift-storage-overview
+- Name: Understanding ephemeral storage for MicroShift
+  File: understanding-ephemeral-storage-microshift
+- Name: Understanding persistent storage for MicroShift
+  File: understanding-persistent-storage-microshift
+- Name: Configuring persistent storage for MicroShift
+  Dir:  persistent_storage_microshift
+  Distros: microshift
```

Comment:
You've already identified the entirety of the "Storage" book as distros: microshift, so you don't need to do it for all of these subdirectories too.